Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Pandaria - Jinyu Portraits

## Changelog:
- Added functional Jinyu portraits.

## How to test:
- Verify if the portraits appear without problems and check whether the clothing is assigned properly.
